Dockerfile.template

```diff
@@ -65,7 +65,8 @@ RUN echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "tools" && \
 	echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "platform-tools" && \
 	echo y | ${CMDLINE_TOOLS_ROOT}/sdkmanager "build-tools;33.0.2"
```

Comment:
Is this the correct build-tools version?
I would also check gradle and gcloud versions